src/main/resources/temp-application.properties

```diff
@@ -0,0 +1,31 @@
+server.port=8091
```

Comment:
please remove this

Reply:
remove, meaning completely delete file? or just the 1st line?

Reply:
completely, what do you do with this file? is this used somewhere? why would need a such "temp" file